package.json

```diff
@@ -14,9 +14,15 @@
     "deploy": "sh scripts/deploy.sh",
     "mocha": "sh scripts/mocha.sh"
   },
+  "resolutions": {
+    "relay-compiler": "https://github.com/alloy/relay/releases/download/v1.5.0-plugin.2/relay-compiler-1.5.0-plugin.2.tgz",
+    "relay-runtime": "https://github.com/alloy/relay/releases/download/v1.5.0-plugin.2/relay-runtime-1.5.0-plugin.2.tgz",
+    "babel-plugin-relay": "https://github.com/alloy/relay/releases/download/v1.5.0-plugin.2/babel-plugin-relay-1.5.0-plugin.2.tgz",
+    "react-relay": "https://github.com/alloy/relay/releases/download/v1.5.0-plugin.2/react-relay-1.5.0-plugin.2.tgz"
+  },
```

Comment:
Note that these items can go away once @alloy's recent Relay work has been merged into master.